Week1/Homework/type.js

```diff
@@ -0,0 +1,40 @@
+let bird = 'canary';
+let dog = 'pug';
+let zoo = { name: 'elephant', age: '15' };
+let pet = { name: 'cat', color: 'grey', length: '40cm' };
```

Comment:
👍 for logic but please try to use `const` instead of `let` wherever you are not re-assigning variables. It makes code easy to understand 🙂